app/_hub/kong-inc/forward-proxy/index.md

```diff
@@ -1,6 +1,6 @@
 ---
 
-name: EE Forward Proxy
+name: Enterprise Forward Proxy
```

Comment:
At one point, there was a directive not to include references to Core or Enterprise in plugin names. Hence, names like "Rate Limiting Advanced" (https://docs.konghq.com/hub/kong-inc/rate-limiting-advanced/) and "LDAP Auth Advanced" (https://docs.konghq.com/hub/kong-inc/ldap-auth-advanced/).

In light of this, should this be "Forward Proxy Advanced"? Or, if there's no corresponding forward proxy plugin in CE (acronym used for clarity), simply "Forward Proxy"?